components/codec/src/byte.rs

```diff
@@ -112,6 +159,18 @@ impl MemComparableByteCodec {
         }
     }
 
+    /// Performs in place bitwise NOT for specified memory region, without doing bounds checking.
+    #[inline]
+    fn flip_bytes_in_place_unchecked(src: &mut [u8], len: usize) {
+        unsafe {
+            let mut src_ptr = src.as_mut_ptr();
+            for _ in 0..len {
+                *src_ptr = !*src_ptr;
+                src_ptr = src_ptr.add(1);
```

Comment:
How about call https://doc.rust-lang.org/std/primitive.slice.html#method.get_unchecked directly?

Reply:
Got it.

Reply:
get_unchecked_mut instead of get_unchecked.

https://doc.rust-lang.org/std/primitive.slice.html#method.get_unchecked_mut